ideas/150-gas-abstraction.md

```diff
@@ -0,0 +1,92 @@
+# Gas Abstraction
+
+## Preamble
+
+    Idea: 150-gas-abstraction
+    Title: Gas Abstraction
+    Status: In Progress
+    Created: 2018-02-01
+    Requires: 145-identity
```

Comment:
Looking at 145-identity it looks like there's a circular dependency which isn't ideal. Can we separate these out somehow?

Reply:
I guess gas-abstraction is dependent of identity, but identity not.
However in a UX aspect, gas-abstraction enhances Identity.

Reply:
gas-abstraction depends on Identity, but it could use other contract as base.
WIth this base the contract can start paying fee for people calling it with owner determined parameters, which are verified by a signed msg. 
Could be a MultiSigWallet.